sphinxcontrib/rst2myst/writers/accumulators.py

```diff
@@ -100,7 +100,8 @@ def start_list_item(self):
         self.list_item = []
 
     def addto_list_item(self, content):
-        self.list_item.append(content)
+        if self.list_item:
```

Comment:
@AakashGfude what is this condition for?

Reply:
@mmcky on running `make myst` on https://github.com/QuantEcon/lecture-python-advanced . it was throwing error - `cannot append to NoneType` for `self.list_item`. So just checking if the list exists before appending. Is `addto_list_item` function always called after `start_list_item` has been called?

Reply:
not sure -- but will look into it. I think state checking like this should probably be in the `translator` to avoid introduction of possible edge cases. This also seems to alter the `myst` output to fail tests. I will take a closer look. Thanks @AakashGfude

Reply:
aah ok. true should transfer it to the translator where this function is called.